src/clue/test/test_adafruit_clue.py

```diff
@@ -62,18 +62,142 @@ def _send_helper(self, image):
         self.main_img = image
 
     def test_buttons(self):
-        BUTTON_A = "button_a"
-        BUTTON_B = "button_b"
-
-        clue._Clue__update_button(BUTTON_A, True)
+        clue._Clue__update_button(CONSTANTS.CLUE_STATE.BUTTON_A, True)
         assert clue.button_a
-        clue._Clue__update_button(BUTTON_A, False)
+        clue._Clue__update_button(CONSTANTS.CLUE_STATE.BUTTON_A, False)
         assert not clue.button_a
 
-        clue._Clue__update_button(BUTTON_B, True)
+        clue._Clue__update_button(CONSTANTS.CLUE_STATE.BUTTON_B, True)
         assert clue.button_b
-        clue._Clue__update_button(BUTTON_B, False)
+        clue._Clue__update_button(CONSTANTS.CLUE_STATE.BUTTON_B, False)
         assert not clue.button_b
 
         assert set(["A", "B"]) == clue.were_pressed
         assert set() == clue.were_pressed
+
+    def test_acceleration(self):
+        MOCK_MOTION_X_A = 1
+        MOCK_MOTION_Y = 2
+        MOCK_MOTION_Z = 3
+        MOCK_MOTION_X_B = 4

```

Comment:
Why did you decide on this approach as opposed to using `pytest.parametrize`?

Reply:
It slipped my mind 😭 I updated it to use it now. I love pytest.parametrize :D